source/mongoexport.txt

```diff
@@ -705,6 +705,13 @@ Options
    You can sort the results with the :option:`--sort` option to
    :binary:`~bin.mongoexport`.
    
+.. option:: --queryFile=<filename>
+
+   An alternative to :option:`--query <mongoexport --query>`. The
+   :option:`--queryFile <mongoexport --query>` option allows you to
```

Comment:
[nit] We want to avoid linking text to the section it's already in, so this might be better suited with plain monospace.
```suggestion
   ``--queryFile`` option allows you to
```